### ISSUE TITLE
Timer not working when app is close

### ISSUE THREAD

**Issue by tahagit** [ocetnik/react-native-background-timer#119]
Hello

I tried to use this library for my app and it's work successfully! but when i close application completely in ios, background timer not running

can you help me?

**Comment by phipop-asn** [comment]
+

**Comment by denizbas92** [comment]
you  are talking about the killing the app ,arent you ? As I know, it does not work when the app is killed

**Comment by mttirpan** [comment]
@denizbas92 Yes, killing the app. Do you know any solution? I need to access significant location and run axios every minutes

**Comment by denizbas92** [comment]
> @denizbas92 Yes, killing the app. Do you know any solution? I need to access significant location and run axios every minutes

If you are coding react-native for just android,you can use the react-native-background-jop libary, it is working when the app is killed,you can put your gps part inside the method that the library gives you. In Ios part, there is a library called react-native-background-gps and you ca use it. As I know ios part is free.

**Comment by denizbas92** [comment]
If you are coding react-native for just android,you can use the react-native-background-jop libary, it is working when the app is killed,you can put your gps part inside the method that the library gives you. In Ios part, there is a library called react-native-background-gps and you ca use it. As I know ios part is free.

**Comment by beshoo** [comment]
@denizbas92 i am sending the app to background, NOT KILLING, And there is a webview i want to Reload in background each min.

Well, it seems android pause the app completely after 180 sec.

Any advice how to keep app running even when mobile is locked? 

Regards

**Comment by gunduzcihat** [comment]
is there any update about this issue?

**Comment by lyseiha** [comment]
+1

**Comment by winardis** [comment]
same issue here, seems like the package only works when app goes from foreground to background, not the app killed background condition i'm trying to support for my app

**Comment by dipali43** [comment]
any other solution to work in ios background service when app is killed ?

**Comment by humblesami** [comment]
> any other solution to work in ios background service when app is killed ?

Its https://github.com/vikeri/react-native-background-job
that works even when app is closed (not only in background=> (app not is focus) and foreground)

https://npmtrends.com/react-native-background-job-vs-react-native-background-task-vs-react-native-background-timer
